### PR TITLE
Health analyzer percentage fix

### LINKED ISSUE #15124: Simple animals scanned with the health analyzer show bizzare amounts of HP
For example, a blobbernaut with 240 hp, is 240% healthy

### PR DESCRIPTION
**Pull request by kingofkosmos**
Makes the health analyzer show the actual percentage of the analyzed mob's health.

Fixes #15124

### CODE REVIEW

**Comment by kingofkosmos** [comment]
Why did travis fail? Everything compiles fine on my end.

**Comment by KorPhaeron** [comment]
It is just ministation, so you can probably ignore it

**Comment by kingofkosmos** [comment]
I actually just updated my master a few hours ago and did this, and used ministation when compiling as well so it'll compile faster, and it worked. Weird stuff.

**Comment by RemieRichards** [comment]
@kingofkosmos are you aware that:
1. Travis does more than just compilation tests
2. Travis is not a foolproof service
3. The third party programs we need to run tests aren't foolproof (In this case, it failed to install one of the third party programs it needed to do our tests, and the only thing it could do was error, this is why it appears as only 1 error.)

**Comment by optimumtact** [comment]
Also travis isn't like a blank wall that just rejects or approves you, you can click on the job and go to the build site and view the console output of each job

File: code/game/objects/items/devices/scanners.dm [hunk 1]
```diff
@@ -172,7 +172,7 @@ MASS SPECTROMETER
 
 	// Time of death
 	if(M.tod && (M.stat == DEAD || (M.status_flags & FAKEDEATH)))
-		user << "<span class='info'>Time of Death:</span> [M.tod]"
+		user << "<span class='info'>Time of Death: [M.tod]</span>"
```
Comment (KorPhaeron) [inline code/game/objects/items/devices/scanners.dm]: I don't think this needs to be changed

**Comment by kingofkosmos** [comment]
Thanks RR and optimumtact I was not aware of that.

File: code/game/objects/items/devices/scanners.dm (hunk 1 above)
Comment (kingofkosmos) [inline code/game/objects/items/devices/scanners.dm]: The time itself was a black text and everything else was neatly blue. You think that's intentional?

Comment (KorPhaeron) [inline code/game/objects/items/devices/scanners.dm]: To make the result stand out, yeah

**Pull request merged**